builds/e2e/nested-longhaul.yaml

```diff
@@ -17,235 +17,65 @@ stages:
-      nestededge: true     
-    steps:   
-      - template: templates/nested-get-secrets.yaml     
-      - template: templates/lock-test-agent.yaml     
-      - template: templates/nested-get-root-ca.yaml  
-      - template: templates/nested-longhaul-deploy.yaml
-        parameters:
-          release.label: 'lh$(agent.group)'
-          test.buildNumber: '$(Build.BuildNumber)'
-          test.buildId: '$(Build.BuildId)'
-          test.startDelay: '$(test.startDelay)'
-          build.source.branch: '$(Build.SourceBranchName)'
-          build.repo.path: '$(Build.SourcesDirectory)'
-          edgelet.source.branch: '$(edgelet.package.branchName)'
-          images.source.branch: '$(images.branchName)'
-          edgelet.artifact.name: '$(edgelet.artifact.name)'
-          images.artifact.name: '$(az.pipeline.images.artifacts)'
-          container.registry: '$(cr.address)'
-          container.registry.username: '$(edgebuilds-azurecr-io-username)'
-          container.registry.password: '$(edgebuilds-azurecr-io-pwd)'
-          iotHub.connectionString: '$(IotHubStressConnString)'
-          eventHub.connectionString: '$(EventHubStressConnStr)'
-          loadGen.message.frequency: '$(loadGen.message.frequency)'
-          longHaul.desiredModulesToRestartCSV: '$(longHaul.desiredModulesToRestartCSV)'
-          longHaul.restartIntervalInMins: '$(longHaul.restartIntervalInMins)'
-          longHaul.sendReportFrequency: '$(longHaul.sendReportFrequency)'
-          test.runtimeLogLevel: '$(test.runtimeLogLevel)'
-          logAnalytics.workspaceId: '$(kvLogAnalyticWorkspaceId)'
-          logAnalytics.sharedKey: '$(kvLogAnalyticSharedKey)'
-          twinTester.twinUpdateSize: '$(twinTester.twinUpdateSize)'
-          twinTester.twinUpdateFrequency: '$(twinTester.twinUpdateFrequency)'
-          twinTester.twinUpdateFailureThreshold: '$(twinTester.twinUpdateFailureThreshold)'
-          twinTester.edgeHubRestartFailureTolerance: '$(twinTester.edgeHubRestartFailureTolerance)'
-          metricsCollector.metricsEndpointsCSV: '$(metricsCollector.metricsEndpointsCSV)'
-          metricsCollector.scrapeFrequencyInSecs: '$(metricsCollector.scrapeFrequencyInSecs)'
-          metricsCollector.uploadTarget: '$(metricsCollector.uploadTarget)'
-          metricsCollector.hostPlatform: '$(hostPlatform)'
-          longHaul.parentHostname: '$(parentName)'
-          longHaul.parentEdgeDevice: '$(parentDeviceId)'
-          testResultCoordinator.storageAccountConnectionString: '$(EdgeLonghaulStorageAccountConnString)'
-          quickstart.package.name: '$(quickstart.package.name)'
-          aziotis.artifact.name: '$(aziotis.artifact.name)'
-          aziotis.package.filter: '$(aziotis.package.filter)'
+    parameters:
+      transportType: amqp
+  - template: templates/nested-longhaul-deploy-amd64.yaml
+    parameters:
+      transportType: amqp
+  - template: templates/nested-parent-vm-setup.yaml
```

Comment:
We discussed this offline. It would require a refactor to the template yamls to make them into steps, not jobs, that is not worth it at the moment. AMQP/MQTT is already parallelized so we don't gain too much anyway